libcaf_core/caf/chunk.hpp

```diff
@@ -166,6 +166,13 @@ class CAF_CORE_EXPORT chunk {
     return std::move(data_);
   }
 
+  // -- comparison -------------------------------------------------------------
+
+  bool equal_to(const chunk& other) const noexcept {
```

Comment:
This needs `#include <algorithm>`. Please push the implementation to the `.cpp` file to avoid including that header here.